src/include/zf_internal/attr_tmpl.h

```diff
@@ -325,6 +325,26 @@ ZF_ATTR(int, force_separate_tx_vi, beta, 0, NULL,
         "The feature is unstable."
         )
 
+ZF_ATTR(str, rx_datapath, stable, "express", NULL,
+        "zf_vi",
+
+        "Options for the rx_datapath mode to use. Set to:\n"
+        " 'express' for low-latency mode;\n"
+        " 'enterprise' for full-feature mode;\n")
+
+ZF_ATTR(int, phys_addr_mode, stable, 0, NULL,
+        "zf_vi",
+
+        "This option affects how DMA buffers are managed.  The default packet buffer \n"
+        "mode uses a limited hardware resource, and so restricts the total amount \n"
+        "of memory that can be used by TCPDirect for DMA. \n"
+        "\n"
+        " 1 - Physical address mode. Inherently unsafe; no address space "
+        "separation between different stacks or net driver packets."
+        "\n"
+        "0 - Virtual address mode."
```

Comment:
We are supposed to be presenting this as effectively a bool. Perhaps replace this doc section with something like:
```suggestion
This option enables physical addressing mode. This makes the DMA mapped addresses directly visible to user space, which should only be used where applications are trusted. The sfc_char module parameter phys_mode_gid can be used to control which users are able to use physical addressing mode.\n
        " 1 - Enable physical addressing mode. Inherently unsafe; no address space "
        "separation between different stacks or net driver packets."
        "\n"
        "0 - Don't enable physical addressing mode. User space sees virtual addresses which are translated by hardware or in the kernel."
```